addons/sale_product_configurator/views/sale_views.xml

```diff
@@ -20,26 +20,16 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//tree/field[@name='product_id']" position="after">
-                <field
-                    name="product_template_id"
-                    string="Product"
-                    options="{'no_open': True}"
-                    attrs="{
-                        'readonly': [('product_updatable', '=', False)],
-                        'required': [('display_type', '=', False)],
-                    }"
-                    context="{
-                        'partner_id': parent.partner_id,
-                        'quantity': product_uom_qty,
-                        'pricelist': parent.pricelist_id,
-                        'uom':product_uom,
-                        'company_id': parent.company_id,
-                        'default_lst_price': price_unit,
-                        'default_description_sale': name
-                    }"
-                    widget="product_configurator"
-                />
+            <xpath expr="//tree/field[@name='product_template_id']" position="attributes">
+                <attribute name="attrs">
```

Comment:
These `attrs` can go back to the sale module I think.

Reply:
not the invisible=0 for the product_template ^^. but the rest yes.
Ty :).

Reply:
That's why I said `attrs` yes :D

Reply:
reading and answering too fast, mb :man_facepalming: